documentation/book/proc-changing-a-topic.adoc

```diff
@@ -42,18 +42,18 @@ CAUTION: Increasing `spec.partitions` for topics with keys will change how recor
 . Update the `KafkaTopic` resource in {ProductPlatformName}.
 +
 ifdef::Kubernetes[]
-In {KubernetesName} this can be done using `kubectl`
+In {KubernetesName} this can be done using `kubectl replace`
```

Comment:
`kubectl apply`? How does actually replace behave? It AFAIK doesn't patch the resource so this might be potentially very dangerous, or? Or is it guaranteed that replace fires only UPDATE event?

Reply:
Well we agree to switch to using `apply` more generally, so let's just do that.